content/intro-to-storybook/vue/es/deploy.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Desplegar Storybook'
+tocTitle: 'Desplegar'
+description: 'Desplegar Storybook online con GitHub y Netlify'
+---
+
+En este tutorial hemos ejecutado Storybook en nuestra máquina de desarrollo. También se puede compartir ese Storybook con el equipo, especialmente con los miembros no técnicos. Afortunadamente, es fácil implementar Storybook en línea.
+
+<div class="aside">
+<strong>¿Hiciste los test con Chromatic antes?</strong>
+<br/>
+🎉 Sus historias ya están desplegadas! Chromatic indexa de forma segura sus historias en línea y las rastrea a través de ramas y commits. Salta este capítulo y ve a la <a href="/react/es/conclusion">conclusión</a>.
+</div>
+
+## Exportando como una app estática
+
+Para desplegar Storybook primero necesitamos exportarlo como una aplicación web estática. Esta funcionalidad ya está incorporada en Storybook, sólo necesitamos activarla añadiendo un script al `package.json`.
+
+```javascript
+{
+  "scripts": {
+    "build-storybook": "build-storybook -c .storybook -s public -o storybook-static"
```

Comment:
this script needs to be updated. If not mistaken the only thing we need here is the -s flag to grab the public folder, the others are not required correct @domyen or @tmeasday ? If this is correct and as you're already made progress with the translation i think that this can updated and the rest of the docs will follow suit shortly